srcpkgs/gamemode/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'gamemode'
+pkgname=gamemode
+version=1.5.1
+revision=1
+build_style=meson
+configure_args="-Dwith-systemd=false -Dwith-dbus-service-dir=/usr/share/dbus-1/services --prefix=/usr --buildtype debugoptimized"
+hostmakedepends="git cmake meson ninja pkg-config"
+makedepends="inih-devel dbus-devel elogind-devel"
+short_desc="Daemon/lib combo for optimisations running games"
+maintainer="Hyper-KVM <hyperkvmx86@gmail.com>"
+license="BSD 3-Clause License (Revised)"
+homepage="https://github.com/FeralInteractive/gamemode"
+distfiles="https://github.com/FeralInteractive/gamemode/archive/${version}.tar.gz"
+checksum="c510fd1391263c6fa05570de840865b1f0c8d573a7acbf488027327ae5a3bbde"
+
+post_install() {
+	cd ${wrksrc}
+	vdoc example/gamemode.ini
+	vlicense LICENSE.txt
+}
+
+depends="lib${pkgname}>=${version}_${revision} polkit"
```

Comment:
This should not be in the middle of the template, this belongs below `makedepends`, also be explicit instead of constructing something like `lib${pkgname}` -> `libgamemode`.